pages/common/mosh.md

```diff
@@ -0,0 +1,33 @@
+# mosh
+
+> Mobile Shell (`mosh`) is remote terminal application that allows roaming.
+> `mosh` is a more robust and responsive replacement for SSH.
+> Visit <https://mosh.org> for more information.
+
+- Typical usage:
```

Comment:
Hi @agnivade, I thought that the sentence "`mosh` persists connections to remote servers while roaming between networks." addresses the point brought up by @waldyrious, while also promoting the capability of `mosh` to seamlessly persist connections to the remote server even when local connectivity is affected (e.g. loss of internet connectivity).

Would you suggest that we alter this statement any further?

Thank you!